t/t1007-kvs-lookup-watch.t

```diff
@@ -143,6 +143,28 @@ test_expect_success NO_CHAIN_LINT 'flux kvs get --watch and --uniq do not see du
 	test_cmp expected seq4.out
 '
 
+#
+# waitcreate tests
+#
+
+test_expect_success 'flux kvs get --waitcreate works on existing key' '
+	flux kvs put test.waitcreate1=1 &&
+	run_timeout 2 flux kvs get --waitcreate test.waitcreate1 >waitcreate1.out &&
+	echo 1 >waitcreate1.exp &&
+	test_cmp waitcreate1.exp waitcreate1.out
+'
+
+test_expect_success 'flux kvs get --waitcreate works on non-existent key' '
+        ! flux kvs get test.waitcreate2 &&
+        flux kvs get --waitcreate test.waitcreate2 > waitcreate2.out &
+        pid=$! &&
+        wait_watcherscount_nonzero primary &&
+        flux kvs put test.waitcreate2=2 &&
+	$waitfile --count=1 --timeout=10 \
```

Comment:
Shouldn't thiss just be a `flux kvs get --waitcreate test.waitcreate`?

Reply:
Ahhh, I see, use that instead of waitfile, but still do the check on the prior get.  Good idea.

Reply:
oh wait, I need the waitfile call to wait on the background process to complete, otherwise the test can be racy.  Or were you thinking of something else?

Reply:
ahh good catch, I'll add that.  It's sort of covered with the --watch & --waitcreate tests, but it's good to cover that too.